doc/sphinx/requirements.txt

```
@@ -0,0 +1,13 @@
+altair>=5.0
+breathe>=4.30
+myst-parser[linkify]>=0.14.0
+sphinx-hoverxref>=0.3b1
+sphinx-design
+sphinx>=5.3,<6
+sphinx_rtd_theme
```

we want to use the `sphinx_book_theme` not the `rtd` one.
```suggestion
sphinx_book_theme
```